### PR TITLE
Prediction fix

### PR DESCRIPTION
**Pull request by jcoll3**
make_prediction did not work for svm by default. I also added a simple function to print the accuracy. The scores were not being used to I removed them.

### CODE REVIEW

File: HW3/classifier.py [hunk 1]
```diff
@@ -18,13 +19,18 @@ def train_classifier(features, labels, args):
 
 
 def make_predictions(clf, features):
-    prediction_probs = clf.predict_proba(features)
-    return np.argmax(prediction_probs, axis=-1) + 1, \
-        np.max(prediction_probs, axis=-1)
+    return clf.predict(features)
```
Comment (erfannoury) [inline HW3/classifier.py]: Scores could be used for finding the most cat-like dog, the most dog-like cat, and the most dog-like dog and the most cat-like cat (for visualizing false predictions).

Comment (jcoll3) [reply HW3/classifier.py]: True, I haven't gotten to that part of the assignment yet. In that case  it needs 
clf = SVC(C=args.svm_lambda, probability=True)

**Comment by jcoll3** [comment]
Should be good now.

File: HW3/classifier.py (hunk 1 above)
Comment (erfannoury) [reply HW3/classifier.py]: Yes, you're right.

**Comment by erfannoury** [comment]
Ok, LGTM.

**Comment by erfannoury** [comment]
Merged, thanks!

**Pull request merged**